ckanext/iati_generator/templates/package/read_base.html

```diff
@@ -0,0 +1,4 @@
+{% ckan_extends %}
+
+{% snippet 'package/snippets/package_iati.html' %}
```

Comment:
Here we just need to add the new tab.

This will be something like this (check and test)

```suggestion

{% block content_primary_nav %}

    {{ super() }}

    {% if h.check_access('sysadmin')) %}
      {{ h.build_nav_icon('iati_generator.iati_page', _('IATI'), package_id=pkg.id, icon='lock') }}
    {% endif %}

{% endblock %}
```